pkgs/applications/science/logic/potassco/clingcon.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, bison, re2c }:
+
+stdenv.mkDerivation rec {
+  pname = "clingcon";
+  version = "3.3.0";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "potassco";
+    repo = "${pname}";
+    rev = "d36fd87c8937b0ae2b39593aab26f3eb717749d3";
```

Comment:
Why don't you use `version` (without quotes) or `v${version}` (or what ever upstream uses to tag it's releases)?